PKGBUILD

```diff
@@ -38,5 +44,8 @@ package() {
 
   cd "${srcdir}/${pkgname}-${pkgver}"
   python setup.py install --prefix=/usr --root="${pkgdir}" --optimize=1
+  
   install -D -m644 COPYING "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
+  install -D -m644 "${srcdir}/${pkgname}.desktop" "${pkgdir}/usr/share/applications/${pkgname}.desktop"
+  install -D -m644 "${srcdir}/${pkgname}-${pkgver}/data/${pkgname}_halo.svg" "${pkgdir}/usr/share/pixmaps/${pkgname}.svg"
```

Comment:
I have around 20 icons in `/usr/share/pixmaps/`, while there are way over 100 in `/usr/share/icons/hicolor/32x32/apps/`. While the [Icon Theme Specification](https://standards.freedesktop.org/icon-theme-spec/icon-theme-spec-0.13.html) expicitly mentions `pixmaps`, all gnome apps that I checked put their icons into `icons`. AppStream also [advises against using `pixmaps`](https://wiki.debian.org/AppStream/Guidelines#Icon_best_practices).

But the main reason is that the pacman [hook that updates the icon cache](https://git.archlinux.org/svntogit/packages.git/tree/trunk/gtk-update-icon-cache.hook?h=packages/gtk3) only watches the latter path. I changed the path in my PR in ae04c8ad24cef243a74c1563a49d16b2a7171787.